runtime/module.go

```diff
@@ -271,7 +271,7 @@ func ProvideEnvironment(
 	return kvService, memStoreService, NewEnvironment(
 		kvService,
 		logger.With(log.ModuleKey, fmt.Sprintf("x/%s", key.Name())),
-		EnvWithRouterService(queryServiceRouter, msgServiceRouter),
+		EnvWithRouterService(queryServiceRouter, msgServiceRouter, map[string]bool{}), // TODO(@julienrbrt): add allow list in app config
```

Comment:
this allow list is kind of like the blacklist/module accounts in app.go. It's a pretty poor ux and we have had issues to remove them for years. Im not sure another allowlist is the way to go